03_how_ln_works.asciidoc

```diff
@@ -721,9 +721,9 @@ When settled on-chain, it must be settled in "satoshis".
 Bitcoin and Lightning have different scarce resources. 
 The scarce resource for operating on the Bitcoin protocol is the space in the blockchain. 
 The scarce resource for operating on Lightning is the channel capacity, or in other words the liquidity.
-Hence, the two different pricing models, each one according to its scarcity. 
-Bitcoin transaction fees pay for block space. 
-Lightning transaction fees pay for channel capacity. 
+Hence, the two different pricing models, each one influenced by its scarcity. 
+Bitcoin transaction fees pay primarily for block space. 
+Lightning transaction fees pay by and large for channel capacity. 
```

Comment:
I am not even sure if I understand the grammer of the addition `by and large`

Reply:
This is just proper English. "by and large" is identical in meaning to "primarily". A synonym. See https://en.wiktionary.org/wiki/primarily.

Reply:
Took out the last 3 lines. As you said it is more about economy than about technology.

See commit # 3 in 74cdd1f0d47cef9b25ad884d851ecadd8ed1f00b